flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/utils/PlannerMocks.java

```diff
@@ -96,6 +102,7 @@ private PlannerMocks(
                 true,
                 ExpressionResolver.resolverFor(
                         tableConfig,
+                        Thread.currentThread().getContextClassLoader(),
```

Comment:
Why not use `PlannerMocks.class.getClassLoader()` here?

Reply:
In test classpath, this shouldn't make any difference. Anyway, I changed it

Reply:
It's more for consistency, and avoid confusion, if someone checks the code and wonders why here is different.